tardis/montecarlo/spectrum.py

```diff
@@ -2,14 +2,14 @@
 import numpy as np
 from astropy import constants, units as u
 
-from tardis.io.util import to_hdf
+from tardis.io.util import HDFReaderWriter
 
-
-class TARDISSpectrum(object):
+class TARDISSpectrum(object, HDFReaderWriter):
     """
     TARDIS Spectrum object
     """
-
+    hdf_properties = ['luminosity_density_nu', 'delta_frequency', 'wavelength',
```

Comment:
The `hdf_properties` *should* be identical to the arguments of `__init__`. However this does not work with this version of the spectrum class, hence why I suggested using the PR version.